solutionbox/structured_data/datalab_solutions/structured_data/_package.py

```diff
@@ -230,16 +222,17 @@ def local_train(preprocessed_dir, transforms_config_file, output_dir,
   print('Local training done.')
 
 
-def cloud_train(preprocessed_dir, transforms_config_file, output_dir,
-                staging_bucket,
+def cloud_train(preprocessed_dir, schema_file, transforms_file, output_dir,
+                model_type, staging_bucket,
```

Comment:
stage-bucket has to be a real bucket (gs://bucket) and not a path (gs://bucket/folder). I would rather ask the user, otherwise they might think "why, are there all these files in my bucket, who made them?"